src/test/unit/lambda_tracer/test_tracer.py

```diff
@@ -186,30 +185,28 @@ def lambda_test_function(event, context):
     assert Configuration.should_report == "123"
 
 
-def test_wrapping_print_happy_flow(context):
+def test_wrapping_print_happy_flow(context, capsys):
     @lumigo_tracer()
     def lambda_test_function(event, context):
         print("hello")
         return 1
 
-    with CaptureOutput() as capturer:
-        assert lambda_test_function({}, context) == 1
-        assert any(line == "hello" for line in capturer.get_lines())
+    assert lambda_test_function({}, context) == 1
+    assert "hello" in capsys.readouterr().out
 
 
-def test_wrapping_enhanced_print_backward_compatible(context):
+def test_wrapping_enhanced_print_backward_compatible(context, capsys):
     @lumigo_tracer(enhance_print=True)
     def lambda_test_function(event, context):
         print("hello")
         return 1
 
-    with CaptureOutput() as capturer:
-        assert lambda_test_function({}, context) == 1
-        assert any(line == "hello" for line in capturer.get_lines())
+    assert lambda_test_function({}, context) == 1
+    assert "hello" in capsys.readouterr().out
 
 
 @pytest.mark.parametrize("is_verbose", [True, False])
-def test_skip_collecting_http_parts(monkeypatch, context, is_verbose):
+def test_skip_collecting_http_parts(wrap_all_libraries, monkeypatch, context, is_verbose):
```

Comment:
This is an unrelated change - I noticed that this test doesn't pass if you run it by itself (it passed due to a dependency leak from another test). I just made sure that it'll have all its dependencies.